src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -262,12 +264,18 @@ private void ingestPockets() {
 
                     this.loginRestClient.login(retailUser.getExternalId(), retailUser.getExternalId());
                     userContextPresentationRestClient.selectContextBasedOnMasterServiceAgreement();
-                    this.pocketsConfigurator.ingestPockets(legalEntity.getExternalId());
+                    List<Pocket> createdPockets = this.pocketsConfigurator.ingestPockets(legalEntity.getExternalId());
+                    ingestPocketTransactions(PocketsConfigurator.EXTERNAL_ARRANGEMENT_ORIGINATION_1, createdPockets);
                 }
             });
         }
     }
 
+    private void ingestPocketTransactions(String parentPocketArrangementId,
```

Comment:
This one-line method can be inlined on line 268.

Reply:
removed